src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,72 @@
+$dayWidth: 100px;
+$dayGap: 1px;
+
+body {
+  margin: 0;
+}
```

Comment:
move to the reset file
```suggestion
```

Reply:
you can delete it from this file as you have added this code to the reset file